datatypes/src/primitives/spatial_partition.rs

```diff
@@ -167,23 +167,23 @@ impl SpatialPartition2D {
     }
 
     fn contains_x(&self, other: &Self) -> bool {
-        crate::util::ranges::value_in_range(
+        crate::util::ranges::value_in_range_inclusive(
```

Comment:
We had a discussion in the slack channel. `contains_x` and `contains_y` are solely used by the contains(s:SpatialPartition2D). And in those cases using inclusive is correct since both exclude the lower and right border.